locations/spiders/hilton.py

```diff
@@ -1,13 +1,22 @@
+from pathlib import Path
+from urllib.parse import urlparse
+
 import geonamescache
 import scrapy
+from scrapy.spiders import SitemapSpider
 
 from locations.structured_data_spider import StructuredDataSpider
+from locations.user_agents import CHROME_LATEST
 
 
-class HiltonSpider(scrapy.spiders.SitemapSpider, StructuredDataSpider):
+class HiltonSpider(SitemapSpider, StructuredDataSpider):
     name = "hilton"
     sitemap_urls = ["https://www.hilton.com/sitemap.xml"]
-    download_delay = 0.2
+    custom_settings = {
+        "USER_AGENT": CHROME_LATEST,
+        "DOWNLOAD_DELAY": 0.2,
+    }
+    visited_pages = []
```

Comment:
Can you switch to a set() instead of array to do `in` checks faster.

Reply:
Thank you. Changed.